src/index.js

```diff
@@ -3,13 +3,25 @@ import ReactDOM from "react-dom";
 import "./index.css";
 import App from "./App";
 import { makeServer } from "./server";
+import { BrowserRouter } from "react-router-dom";
+import { WatchLaterProvider } from "./Context/WatchLaterContext";
+import { LikedVideosProvider } from "./Context/LikedVideosContext";
+import { VideoProvider } from "./Context/videoContext";
```

Comment:
Create an index.js file in context folder and import all these contexts in a single file. Then you can easily import all the contexts here easily